hieradata/clients/azure.ci.jenkins.io.yaml

```diff
@@ -27,7 +27,6 @@ profile::buildmaster::letsencrypt: true
 profile::buildmaster::docker_image: 'jenkins/jenkins'
 profile::buildmaster::docker_tag: 'lts-jdk11'
 profile::buildmaster::groovy_init_enabled: true
-profile::buildmaster::groovy_d_agent_security: 'present'
```

Comment:
Shouldn’t this be absent so the file gets deleted? (Not a puppet expert may be wrong)

Reply:
Only if the block `file {}` logic is kept, to handle the parameter.
In this change request, since the code is removed, then the hieradata attribute would not be used.

=> It means that we have to operate 1 time on the controller VMs (ci.j, trusted.j and cert-ci.j) to delete the file on their file system once the change is applied.